workflows4s-quartz/src/test/scala/workflows4s/runtime/wakeup/quartz/QuartzKnockerUpperTest.scala

```diff
@@ -23,11 +24,27 @@ class QuartzKnockerUpperTest extends AnyFreeSpec with Matchers with BeforeAndAft
     "should schedule a wakeup at a specified time" in withQuartzKnockerUpper { knockerUpper =>
       var wokenUpAt: Instant = null
       val testId             = "test-id"
-      knockerUpper.initialize(id => IO { if (id == testId) wokenUpAt = Instant.now() }).unsafeRunSync()
-      val wakeupAt           = Instant.now().plusMillis(100)
+      knockerUpper
+        .initialize(id =>
+          IO {
+            if (id == testId) {
+              wokenUpAt = Instant.now()
+              println(s"Woken up at $wokenUpAt") // Add debug logging
```

Comment:
Lets use `logger` to have a consistent output. It doesn't matter much and it's not that common to log from tests, but I think it makes sense. 

Just extend `StrictLogging` (should already be available on the classpath) and do `logger.info`

Reply:
Working on this one. Will update the PR in a few minutes.